src/rhsmlib/dbus/server.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, bus_class=None, bus_name=None, object_classes=None, bus_kwarg
             self.objects.append(clazz_instance)
             self.object_map[str(clazz.__name__)] = clazz_instance
 
-        consumer_dir_list = [self.identity.reload]
+        consumer_dir_list = []
```

Comment:
This change is not explained. What is this about?

Reply:
`consumer_dir_list` should be empty list. It was caused some strange behavior.

Reply:
Is this the fix that fixed my "out-of-date status without auto-attach" observation?